.github/workflows/production-run_check.yml

```diff
@@ -0,0 +1,73 @@
+name: production run check
+
+on:
+  push:
+    branches: [$default-branch]
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out Github repository
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Cache Docker layers
+        uses: actions/cache@v3
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Build transcendence-app docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: ./transcendence-app
+          target: production
+          push: false
+          load: true
+          tags: ft_transcendence/transcendence-app-prod:latest
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new,mode=max
+      - name: Build swagger docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: ./transcendence-app
+          target: swagger
+          push: false
+          load: true
+          tags: ft_transcendence/swagger:latest
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new,mode=max
+      - name: Generate code
+        run: make gen
+      - name: Build webapp docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: ./webapp
+          target: production
+          push: false
+          load: true
+          tags: ft_transcendence/webapp-prod:latest
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new,mode=max
+      - name: Run
+        run: |
+          docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d
+          docker-compose -f docker-compose.yml -f docker-compose.prod.yml down
```

Comment:
Here we use `docker-compose up` without the `--build` flag to use the images from the cache.

Reply:
what about we add some make rule to stop the production server, analogous to `make prod`? Something like this `docker-compose -f docker-compose.yml -f docker-compose.prod.yml down` maybe?

Reply:
Done :+1: